server/player/player.go

```diff
@@ -246,6 +246,12 @@ func (p *Player) SendJukeboxPopup(a ...any) {
 	p.session().SendJukeboxPopup(format(a))
 }
 
+// SendToast sends a toast to the player. This toast is shown at the top of the screen, similar to achievements or pack
+// loading.
+func (p *Player) SendToast(title, message string) {
```

Comment:
Do you reckon we could come up with a nicer name than "toast"? It's such a shit name lmao.

Reply:
> Do you reckon we could come up with a nicer name than "toast"? It's such a shit name lmao.

Perhaps alert?